prometheus-pusher/main.go

```diff
@@ -115,7 +115,7 @@ func (p *prometheusSink) Sink(ctx context.Context, datumStreamCh <-chan sinksdk.
 		failed = failed.Append(sinksdk.ResponseFailure(datum.ID(), "failed to push the metrics"))
 	}
 	var pls []PrometheusPayload
-	var prometheusPayload PrometheusPayload
+	var prometheusPayloads []*PrometheusPayload
 	for _, payloadMsg := range payloads {
 		p.metrics.IncreaseTotalPushed()
 		if p.enableMsgTransformer {
```

Comment:
Having a feature flag to support univariate payload  and multivariate numalogic payload

Reply:
This is just temporary right?

Reply:
yes. once we moved to Multivariant. we can remove this code